grails-app/controllers/au/org/emii/aatams/detection/DetectionController.groovy

```diff
@@ -64,15 +68,10 @@ class DetectionController extends ReportController
             ValidDetection.get(it.detection_id)
         }
 
-        def paramsClone = params.clone()
-
-        paramsClone.max = grailsApplication.config.filter.count.max + 1
-        def count = detectionExtractService.getCount(paramsClone)
-
         params.remove("sql")
         params.remove("projectPermissionCache")
 
-        [results: detections, count: count]
+        [results: detections, count: -1]
```

Comment:
Did you try just leaving this count value out altogether? I think that would be preferable if it would work.